jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarFiles.java

```diff
@@ -30,16 +35,27 @@
    * Generates a {@link JibContainerBuilder} from contents of a jar file.
    *
    * @param processor jar processor
+   * @param jarOptions jar cli options
+   * @param commonCliOptions common cli options
+   * @param logger console logger
    * @return JibContainerBuilder
    * @throws IOException if I/O error occurs when opening the jar file or if temporary directory
    *     provided doesn't exist
    * @throws InvalidImageReferenceException if the base image reference is invalid
    */
-  public static JibContainerBuilder toJibContainerBuilder(JarProcessor processor)
+  public static JibContainerBuilder toJibContainerBuilder(
+      JarProcessor processor,
+      Jar jarOptions,
+      CommonCliOptions commonCliOptions,
+      ConsoleLogger logger)
       throws IOException, InvalidImageReferenceException {
 
-    // Use distroless as the base image.
-    JibContainerBuilder containerBuilder = Jib.from("gcr.io/distroless/java");
+    // Use distroless as the default base image.
```

Comment:
I think here we should try to check if the jar is java8 or java11 and chose the right distroless? (perhaps in a followup PR)

Reply:
Yup, good point - we have an issue for this #2871. Will make this change in the next PR.